docs/pages/docs/providers/node.md

```diff
@@ -43,15 +44,20 @@ All dependencies found in `package.json` are installed with either NPM, Yarn, PN
 
 The build script found in `package.json` if it exists.
 
-Or, if it's an NX Monorepo (detected if `nx.json` existis), the `build` pipeline for the `NIXPACKS_NX_APP_NAME` app will be called. Otherwise, it will run build for the `default_project` in `nx.json`. The build command is `(npm|pnpm|yarn|bun) run build <NxAppName>:build:production`.
+- Or, if it's a [moon repo](https://moonrepo.dev/moon) (detected if `.moon/workspace.yml` exists), the `build` task for the `NIXPACKS_MOON_APP_NAME` will be called. The task name can be customized with `NIXPACKS_MOON_BUILD_TASK`. This will run the command `moon run <app_name>:<build_task>`.
```

Comment:
Can the Moon docs please be moved below Nx and Turbo repo

Reply:
Did it alphabetical, but sure.